### PR TITLE
Clarifications for bigquery connector

### PR DESCRIPTION
**Pull request by vlasenkoalexey**
- Added code snippet describing how to get better performance when reading data from BQ with large batch sizes or wide rows
- Clarified meaning of the requested_streams argument

### CODE REVIEW

File: tensorflow_io/bigquery.md [hunk 1]
```diff
@@ -95,7 +103,28 @@ if __name__ == '__main__':
 
 ```
 
-It also supports reading BigQuery column with repeated mode (each field contains array of values with primitive type: Integer, Float, Boolean, String, but RECORD is not supported). In this case, selected_fields needs be a dictionary in a
+In some cases when when row is too wide (has 30+ columns) or if you are doing batching with large batch sizes it might be benefitial to do batching before interleave to get a better performance. Here is an exaple showing how to do that:
```
Comment (kvignesh1420) [inline tensorflow_io/bigquery.md:106]: duplicate `when` in first line.

Comment (kvignesh1420) [reply tensorflow_io/bigquery.md:106]: `benefitial` --> `beneficial` in second line

Comment (kvignesh1420) [reply tensorflow_io/bigquery.md:106]: `exaple` --> `example` in third line

File: tensorflow_io/bigquery.md [hunk 2]
```diff
@@ -95,7 +103,28 @@ if __name__ == '__main__':
...
+streams_ds = tf.data.Dataset.from_tensor_slices(streams)
+dataset = streams_ds.interleave(
+    read_rows,
+    cycle_length=streams_count,
+    num_parallel_calls=streams_count,
+    deterministic=False)
+...
+```
+
+Connector also supports reading BigQuery column with repeated mode (each field contains array of values with primitive type: Integer, Float, Boolean, String, but RECORD is not supported). In this case, selected_fields needs be a dictionary in a
```
Comment (kvignesh1420) [inline tensorflow_io/bigquery.md:127]: The last line is incomplete

File: tensorflow_io/bigquery.md (hunk 1 above)
Comment (vlasenkoalexey) [reply tensorflow_io/bigquery.md:106]: so many typos, thanks for reviewing

File: tensorflow_io/bigquery.md (hunk 2 above)
Comment (vlasenkoalexey) [reply tensorflow_io/bigquery.md:127]: updated

**Review by kvignesh1420 — APPROVED** [review]
LGTM. Thanks @vlasenkoalexey

**Review by yongtang — APPROVED** [review]
LGTM, Thanks @vlasenkoalexey

**Pull request merged**